tutorials/notebooks/example_quick_start_imagenet.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
1. You need to clear the output before commit.
2. I think better to rename it to "example_quick_start_torchvision.ipynb"
3. check that you add a license at the end of the file. (you can copy from other notebooks)